### ISSUE TITLE
Form Submission Failed

### ISSUE THREAD

**Issue by ajramosSDI** [searchdiscovery/client-jnj-askgs-ga4-dl-spec#9]
This event should fire whenever a form submission fails i.e. wrong email format, empty required cells, etc.

[Form Submission Succeeded.md](https://github.com/searchdiscovery/client-jnj-askgs-ga4-dl-spec/blob/main/Data%20Layer%20Events/Form%20Submission%20Succeeded.md)

**Comment by mgarzonjnj** [comment]
The submit a request form can be accessed via this link 

https://jnjgsportal--predevesp.sandbox.my.site.com/s/submit-request?language=en_US

**Comment by mandyjohnston03** [comment]
How can we force a fail after preview?

**Comment by mgarzonjnj** [comment]
A fail doing preview is something that would only happen during Salesforce being down/very rare scenario; I don't think we should invest the time in validating this since the likelihood of it happening is very low.  Thoughts Mandy? @mandyjohnston03

**Comment by mandyjohnston03** [comment]
@mgarzonjnj I think it's okay to leave out for now since form format wasn't a large concern during discovery - this was more of a nice to have. However, I would bump this up in priority IF after the data starts coming in, you see a large dropoff between form start and form submit.

**Comment by mgarzonjnj** [comment]
@mandyjohnston03 Sorry, while we can't test a fail for submit a request, we can test the fail for LiveChat.  Though the same bug that we have open for Form Submission Succeeded (Form Category & Form Region not being populated) will also apply to this event and I've already raised it for @nkuma173 to address.  Let's retest LiveChat fail once that is ready.

**Comment by mgarzonjnj** [comment]
@mandyjohnston03 Ready to re-test :)

**Comment by davisbradleyj** [comment]
We will likely need an to have an agent available to force a fail in this scenario for live chat.  If possible this is something we can discuss on Friday (Aug 18).

**Comment by davisbradleyj** [comment]
@mgarzonjnj 

Our team will not be able to test this one, and is something that J&J will need to confirm on their end-to-end testing
